lib/pr_checker.js

```diff
@@ -144,6 +144,13 @@ class PRChecker {
   checkPRWait(now) {
     const { pr } = this;
     const { logger } = this;
+    const labels = pr.labels.nodes.map((l) => l.name);
+    if (labels.includes('code-and-learn')) {
+      logger.info(
+        'This is a Code & Learn PR, it doesn\'t have a time limit.'
```

Comment:
I don't think we need to log anything, to be honest, but maybe others disagree. We already log so much, the less noise, the better.

Reply:
@Tiriel i we should not log anything because we do not log anything about time when it is ready to land and our goal is to have as less noise as possible.